### PR TITLE
quick fixes

### PR DESCRIPTION
**Pull request by simone-silvestri**
Quick fixes as I recklessly merged a _non-test-passing_ PR

### CODE REVIEW

**Review by glwagner — APPROVED** [review]

**Comment by glwagner** [comment]
image

File: src/AbstractOperations/conditional_operations.jl [hunk 1]
```diff
@@ -123,11 +123,11 @@ end
 @inline get_condition(condition::AbstractArray, i, j, k, grid, args...) = @inbounds condition[i, j, k]
 
 
-Base.summary(c::ConditionalOperation) = string("ConditionalOperand of ", summary(c.operand), " with condition ", summary(c.condition))
+Base.summary(c::ConditionalOperation) = string("ConditionalOperation of ", summary(c.operand), " with condition ", summary(c.condition))
 
 Base.show(io::IO, operation::ConditionalOperation) =
     print(io,
-          summary(operation), '\n',
+          "ConditionalOperation at $(location(operation))", '\n',
```
Comment (glwagner) [inline src/AbstractOperations/conditional_operations.jl:130]: why?

Comment (simone-silvestri) [reply src/AbstractOperations/conditional_operations.jl:130]: Just to avoid a bit of repetition

Comment (glwagner) [reply src/AbstractOperations/conditional_operations.jl:130]: I'd argue that rewriting part of `summary` in `show` *is* repetition :-0

Comment (glwagner) [reply src/AbstractOperations/conditional_operations.jl:130]: but don't worry about it

**Pull request merged**